docs/docs/workflows/verify-cluster.md

```diff
@@ -15,6 +15,15 @@ This command performs the following steps:
 2. Verify the signature of the measurements. This will use Edgeless Systems' [public key](https://edgeless.systems/es.pub).
 3. Write measurements into configuration file.
 
+After the command succeeded you will see more entries under the `measurements` key than there are indices in the `enforcedMeasurements` key.
```

Comment:
```suggestion
fter the command succeeded, the configuration file will contain a list of key value pairs in `measurements` and a list of indices in `enforcedMeasurements`.
Not all keys in `measurements` will have a matching index in `enforcedMeasurements`.
```
Not 100% happy with my suggestion, but I think it gets the point across a bit clearer.